src/libraries/System.Security.Cryptography.Algorithms/src/System/Security/Cryptography/CryptoConfig.cs

```diff
@@ -12,7 +12,6 @@
 
 namespace System.Security.Cryptography
 {
-    [UnsupportedOSPlatform("browser")]
```

Comment:
It also needs to be removed from ref, and please remove unused import `using System.Runtime.Versioning;`

Reply:
Thanks!